core/new-gui/src/app/dashboard/component/feature-container/user-dictionary-section/user-dictionary-section.component.html

```diff
@@ -0,0 +1,50 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+<div class="user-dictionary-container subsection-grid-container">
+
+  <mat-card class="button-area subsection-grid-container">
+    <div ngbDropdown class="d-inline-block">
+      <button class="btn btn-outline-primary" id="sortDropdown" ngbDropdownToggle>Sort</button>
+      <div ngbDropdownMenu aria-labelledby="dropdownBasic1">
+        <mat-list>
+          <button mat-button class="sorting_func" (click)="ascSort()">A -> Z</button>
+          <mat-divider></mat-divider>
+          <button mat-button class="sorting_func" (click)="dscSort()">Z -> A</button>
+          <mat-divider></mat-divider>
+          <button mat-button class="sorting_func" (click)="sizeSort()">By Dictionary Size</button>
+          <mat-divider></mat-divider>
+        </mat-list>
+      </div>
+      <span>&emsp;&emsp;&emsp;</span>
+      <button type="button" class="btn btn-outline-dark add-button" (click)="openNgbdModalResourceAddComponent()">
+        <i class="fa">&#xf0ee;</i>
+      </button>
+    </div>
+  </mat-card>
+
+  <mat-card class="user-dictionary-list subsection-grid-container">
+    <mat-card-content>
+      <mat-list>
+        <h2 style="font-family:Roboto, Arial, sans-serif;">Dictionaries</h2>
+        <br />
+        <div class="scroll">
+          <mat-list-item *ngFor="let dictionary of UserDictionary" class="user-dictionary-list-item">
+            <ng-template #tipContent>Description: {{dictionary.description}}</ng-template>
+            <div class="section for tip" placement="bottom" [ngbTooltip]="tipContent">
+              <p font-size=10px><span style="color:rgb(62, 202, 253); font-size:20px; font-family:Roboto, Arial, sans-serif;">
+                  {{dictionary.name}}</span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Item
+                Preview: {{dictionary.items.slice(0,3)}}....
```

Comment:
Rather than using `&nbsp` to represent white spaces, try to add the style `white-space: pre` according to https://stackoverflow.com/questions/35247325/how-to-show-two-continuous-spaces-in-angular